rdm/checklists/62304_2006_AMD1_class_a.txt

```diff
@@ -7,8 +7,8 @@
 include 62304_base_class_a
 include 14971_2007
 # TODO: 4.4 Legacy Software
```

Comment:
I propose that we remove this TODO, and write a comment stating that we don't expect 4.4 to be relevant for anyone using RDM.  Alternatively, we can simply add an audit note to the SDS that states there is no legacy software.

Reply:
I like adding the audit note. It would be a flag for any team that really was trying to use the legacy clause. Where should it go? I am thinking near the front in the `# Purpose` section.

Reply:
Fixed.